storybook/src/components/card/card.css-template.ts

```diff
@@ -64,6 +64,12 @@ export const cssCard: ComponentImplementation<Card<TemplateResult>> = {
                       ${isButtonInterface(interaction) ? buttonTemplate(interaction) : nothing}
                       ${isLabelInterface(interaction) ? labelTemplate(interaction) : nothing}
                       ${isToggletipInterface(interaction) ? toggletipTemplate(interaction) : nothing}
+                      ${isSlideToggleInterface(interaction)
+                        ? html`<dso-slide-toggle
```

Comment:
Waarom hier niet met `slideToggleTemplate()` ?

Reply:
Er is geen HTML/CSS implementatie van Slide Toggle.

Reply:
Uitzoeken templateContainer logica voor bepalen template

Reply:
Ik denk dat je deze hebt gemist, maar je zou nog uitzoeken of je hier `slideToggleTemplate()` kan gebruiken (spoiler: ja).

De code die je ging uitzoeken staat in `template-container.ts`, de instance method `TemplateContainer.create()`. Leesbaar is het niet echt, maar deze functie maakt een instantie van `Templates`. Elk componentTemplate() functie verwijst naar "een" implementatie. Welke dat is, hangt af van hoeveel implementaties er zijn en in welke wereld je zit: als het er maar eentje is, krijg je die. Zijn er twee, dan wordt er naar de preferredImplementation gekeken.

Check it out!

Je vroeg je nog af "Maar waarom mocht ik in dat andere PR dan wel direct een component aanroepen?". Dat had te maken met `<dso-icon icon="spinner">`. De HTML/CSS en Web Component implementaties zijn niet gelijk, de spinner werkt alleen in de Web Component implementatie. We _moesten_ wel expliciet zijn.

List Button is nog zo'n component.

Reply:
✅ slideToggleTemplate() wordt gebruikt